docs/platform-specific/PlatformSpecificAnalyzer.md

```diff
@@ -0,0 +1,52 @@
+---
+title: UWP Platform Specific Analyzer
+author: hermitdave
+description: Platform Specific Analyzer is a Roslyn analyzer that analyzes and suggests code fixes to ensure that any version / platform specific API are guarded by correct runtime checks
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, plaform specific, platform specific analyzer, roslyn analyzer
+dev_langs:
+  - csharp
+  - visualbasic
+---
+
+# Platform Specific Analyzer
+
+The [writing version adaptive](https://docs.microsoft.com/windows/uwp/debug-test-perf/version-adaptive-code) code, the developers should ensure that code checks for presence of API before calling it.
+The platform specific analyzer is a Roslyn Analyzer that can parse through code and suggest fixes where appropriate.
+
+## Installation
+
+The analyzer is available both as a nuget package and also as Visual Studio Extention
```

Comment:
Could not find the Visual Studio Extension, how do we build it?

Reply:
There is a vsix solution and building it is as easy as including it in sln however that throws VSTS errors. Need to get that looked into.

Reply:
Got it. Not familiar with VSIX - what's the benefit over having a nuget package? Should we publish both?

Reply:
I'm sure we can get it building in VSTS, but in general, I don't think it's as good since it requires everyone to install it. Nice part of nuget analyzers is that they are brought in with the solution so clone-open just works.

Reply:
The VSIX will also need to be added to the list of items sent to the signing service.

Reply:
isn't that the reason the analyzer template supports both vsix and nuget ?

Reply:
Sure, but that's a template, doesn't mean you have to support both methods. Most analyzers that I know of just use NuGet.

Reply:
I am okay with whatever you and others think 👍  I have no idea about the sort of usage they have plus VSIX would need submissions to VS gallery etc.. maybe its not worth the effort unless there's a strong demand.

We can go nuget to start with. I will fix the docs